jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/component/JwalaUi.java

```diff
@@ -319,4 +319,14 @@ public void screenShot(final String imgPath) {
             throw new JwalaUiException("Screen shot failed!", e);
         }
     }
+
+    //the dialog box xpaths have been generalized since different popups have different xpaths
+    public void clickOk(){
+        click(By.xpath("//*[text()='Ok']"));
```

Comment:
Can you make this more specific ? Please refrain from using * since you are specifying any element with a text = "Ok". For now it works because there are no other element that has "Ok" text but who knows some element might have one in the future.